packages/esm-forms-dashboard-app/src/hooks/use-forms.ts

```diff
@@ -0,0 +1,141 @@
+import dayjs from 'dayjs';
+import useSWR from 'swr';
+import { getDynamicOfflineDataEntries, openmrsFetch, useConfig, useVisit } from '@openmrs/esm-framework';
+import { ListResponse, Form, EncounterWithFormRef, CompletedFormInfo, FormsSection } from '../types';
+import { customEncounterRepresentation, formEncounterUrl, formEncounterUrlPoc } from '../constants';
+import { ConfigObject, FormsSectionConfig } from '../config-schema';
+
+export function useForms(cachedOfflineFormsOnly = false, patientUuid: string = '') {
+  const { showConfigurableForms, customFormsUrl, showHtmlFormEntryForms } = useConfig() as ConfigObject;
+  const url = showConfigurableForms
+    ? customFormsUrl.concat(`?patientUuid=${patientUuid}`)
```

Comment:
Instead of this `concat()`, it would be better to use the [`interpolateUrl()`](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-config/src/navigation/interpolate-string.ts#L36) function from the framework. The patientUuid could be supplied as an additional property, but that way we're giving people more freedom over the URL they use...